src/context.rs

```diff
@@ -16,21 +16,33 @@ impl MessageContext {
         let res = parse(source).unwrap_or_else(|x| x.0);
 
         for entry in res.body {
+            let id = match entry {
+                ast::Entry::Message{ref id, ..} => {
+                    id.name.clone()
+                }
+                _ => unimplemented!(),
```

Comment:
IIUC, this will panic on Comments or Sections. Is this intended?  The Context doesn't use either of these Entry types.